teamB/eungSooKim/controllers/postControllers.js

```diff
@@ -0,0 +1,65 @@
+const postService = require("../services/postService");
+
+const writePost = async (req, res) => {
+  try {
+    const { title, content, userId } = req.body;
+
+    if (!title || !content || !userId) {
+      return res.status(400).json({ message: "제목과 내용을 입력하세요" });
+    }
+
+    await postService.writePost(title, content, userId);
+    return res.status(201).json({
+      message: "게시되었습니다",
+    });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const postsList = async (req, res) => {
```

Comment:
함수 이름을 지을 때 동상형을 시작하여 어떤 행위를 하는지 나타내는 것이 좋습니다.
다른 부분들은 잘 적용하셨습니다. 
```suggestion
const getPostsList = async (req, res) => {
```